src/view/terminalview.h

```diff
@@ -8,10 +8,10 @@ class TerminalView {
 private:
     std::string formatNumberSpacing(int number, int maxLen);
 public:
-    void render(World world);
-    std::string createTable(World world);
-    std::string stringifyFood(Food food);
-    std::string stringifyEntity(Entity & entity);
+    void render(const World world);
+    std::string createTable(const World world);
+    std::string stringifyFood(const Food food);
```

Comment:
Don't forget the ampersand &!
> (const World & world)
it's important

Reply:
I correct myself:
I think the classes can't be const
if they are const you might not be able to even call methods

Reply:
https://stackoverflow.com/questions/7626116/how-to-pass-object-to-function-in-c
So basically if I don't want to change the original object do nothing and if I don't intend to or it doesn't matter use &?